client/src/GMRoute/GMShared/PlayerListComponents/PlayerMessage/ReminderCreator/index.tsx

```diff
@@ -20,63 +23,62 @@ export function ReminderCreator({
 }: ReminderCreatorProps) {
   const { game } = useDefiniteGame();
   const [, , , setHaseReminder] = usePlayerReminder();
+  const [, , , clearReminder] = useClearPlayerReminder();
   const relevantReminders = useMemo(
-    () => game.reminders.filter(({ name }) => name === reminder),
-    [game.reminders, reminder],
+    () =>
+      game.reminders.filter(({ name, active }) => active && name === reminder),
+    [game, reminder],
   );
   const playerHasReminder = (player: string) =>
     relevantReminders.find((reminder) => reminder.toPlayer === player);
 
   // don't change the ordering when statuses change
-  const frozenPlayerList = useMemo(() => {
-    return [...game.playerList].sort();
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  const frozenPlayerList = game.playerList;
 
   return (
-    <div className="flex flex-col gap-1">
-      <Dialog.Root>
-        <Dialog.Trigger>
-          <Button>Apply {reminder}</Button>
-        </Dialog.Trigger>
+    <Dialog.Root>
+      <Dialog.Trigger asChild>
+        <Button>Apply {reminder}</Button>
+      </Dialog.Trigger>
 
-        <Dialog.Content>
-          <div className="flex flex-col gap-1">
-            <DialogHeader>Applying {reminder}</DialogHeader>
-            <Dialog.Close key="cancel">
-              <Button
-                className="capitalize"
-                size="3"
-                variant="soft"
-                color="lime"
-              >
-                {"Done"}
-              </Button>
-            </Dialog.Close>
-            {frozenPlayerList.map((toPlayer) => (
-              <Button
-                key={toPlayer}
-                className="capitalize"
-                size="3"
-                color="violet"
-                variant={playerHasReminder(toPlayer) ? "soft" : "outline"}
-                onClick={() => {
+      <Dialog.Content>
+        <Dialog.Header>Applying {reminder}</Dialog.Header>
+        <div className="flex flex-1 flex-col gap-1 overflow-y-auto">
+          <Dialog.Close key="cancel" asChild>
+            <Button className="capitalize">{"Done"}</Button>
+          </Dialog.Close>
```

Comment:
Small thing but dialog close key is "cancel" while button itself indicates "done"